modules/ROOT/pages/authentication-authorization/ldap-integration.adoc

```diff
@@ -97,9 +97,24 @@ runtime using xref:reference/procedures.adoc#procedure_dbms_setconfigvalue[`dbms
 
 First, you configure Neo4j to use LDAP as an authentication and authorization provider.
 
-. Uncomment the setting `dbms.security.auth_enabled=false` and change its value to `true` to turn on the security feature.
+. Make sure security is turned on.
```

Comment:
A best practice is to use only words that count, so I wouldn't change step 1. In the case of OpenID Connect, it is "Make sure security is turned on.", because they don't need to do anything, just to verify that it is "true" (which is the default).